fortuna/src/Main.css

```diff
@@ -0,0 +1,89 @@
+.styleForRow {
+    margin-top:20px;
+}
+
+body, html {
+    height: 100%;
+    color: #04CCFF;
+    background: #000921;
+}
+
+h1 {
+    padding: 5px 5px 5px 5px;
+}
+
+
+/* Column Styling */
+.column {
+    float: left;
+    font-family: "Lucida Console";
+}
+
+.left {
+    width: 25%;
+    margin-top: 275px;
+    padding-left: 20px;
+    text-align: justify;
+}
+
+.right {
+    width: 25%;
+    margin-top: 375px;
+    text-align: center;
+}
+
+.middle {
+    width: 50%;
+    text-align: center;
+    margin-top: 350px;
+}
+
+
+/* Button Styling */
+.btn {
+    color: #000921;
+    background: #04CCFF;
+    border: 2px solid #04CCFF;
+    border-radius: 8px;
+    width: 175px;
+}
+
+.btn:hover {
+	color: #04CCFF;
+	background: #000921;
+	border: 2px solid #04CCFF;
+}
+
+.clearbtn {
+    color: #04CCFF;
+    border: none;
+    background: none;
+}
+
+.clearbtn:hover {
+    color: #39a9dd;
+}
+
+.popupbtn {
+    color: #000921;
+    background: #04CCFF;
+    border: 2px solid #04CCFF;
+    border-radius: 8px;
+    width: 150%;
+}
+
+
+/* Popup Styling */
+.popup {
+    padding: 5px 50px 20px 50px;
+    background: #04213F;
+}
+
+/* Leaderboard */
+.leaderboard {

```

Comment:
There will be a leaderboard on the Battle Arena page as well. I can make a separate CSS file for it if everyone thinks it's for the best, but I don't think it is necessary.

Reply:
I definitely think we should make Leaderboard its own component then, that way we can reuse the logic for it. This is a perfect example of things components are good for.

I don't really care if we make it is own CSS file or not. If we do make Leaderboard its own component though, then it will only be used in one place. I'm totally cool if that's something for another PR though.